### PR TITLE
feat(rpc middleware): use custom time buckets

### PR DESCRIPTION
**Pull request by niklasad1**
Closing #11886

### CODE REVIEW

**Review by lexnv — APPROVED** [review]
LGTM!

File: client/rpc-servers/src/middleware.rs [hunk 1]
```diff
@@ -24,6 +24,21 @@ use prometheus_endpoint::{
 	U64,
 };
 
+/// Histogram time buckets in seconds.
+const HISTOGRAM_BUCKETS: [f64; 11] = [
+	5.0,
```
Comment (ggwpez) [inline client/rpc-servers/src/middleware.rs:29]: 5 sec as smallest bucket?  
Not rather milli-seconds? @bakhtin

**Review by ggwpez — CHANGES_REQUESTED** [review]

File: client/rpc-servers/src/middleware.rs (hunk 1 above)
Comment (niklasad1) [reply client/rpc-servers/src/middleware.rs:29]: sorry the comment is wrong it's in micro seconds

**Review by ggwpez — APPROVED** [review]
Someone with RPC experience should probably approve this, to me it looks good.

**Comment by niklasad1** [comment]
bot merge

**Pull request merged**